packages/paste-core/components/keyboard-key/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import { render } from "@testing-library/react";
+import { CustomizationProvider } from "@twilio-paste/customization";
+import { Theme } from "@twilio-paste/theme";
+import * as React from "react";
+
+import { KeyboardKey, KeyboardKeyGroup } from "../src";
+import { Default } from "../stories/index.stories";
+
+describe("KeyboardKey", () => {
+  it("should render", async () => {
+    const { getAllByText } = render(<Default />);
+
+    const controlKey = getAllByText("Control")[0];
+    const bKey = getAllByText("B")[0];
+    expect(controlKey).toBeDefined();
+    expect(controlKey).toBeDefined();
+    expect(controlKey).toHaveAttribute("data-paste-element", "KEYBOARD_KEY");
+    expect(bKey).toHaveAttribute("data-paste-element", "KEYBOARD_KEY");
+    expect(controlKey.parentElement).toHaveAttribute("data-paste-element", "KEYBOARD_KEY_GROUP");
+  });
+
+  describe("Customization", () => {
+    it("should accept custom element names", async () => {
+      const { getAllByText } = render(
+        <Theme.Provider theme="default">
+          <KeyboardKeyGroup element="MY_CUSTOM_KEY_GROUP">
+            <KeyboardKey element="MY_CUSTOM_KEY_ONE">Control</KeyboardKey>
+            <KeyboardKey element="MY_CUSTOM_KEY_TWO">B</KeyboardKey>
+          </KeyboardKeyGroup>
+        </Theme.Provider>,
+      );
+
+      const controlKey = getAllByText("Control")[0];
+      const bKey = getAllByText("B")[0];
+      expect(controlKey).toHaveAttribute("data-paste-element", "MY_CUSTOM_KEY_ONE");
+      expect(bKey).toHaveAttribute("data-paste-element", "MY_CUSTOM_KEY_TWO");
+      expect(controlKey.parentElement).toHaveAttribute("data-paste-element", "MY_CUSTOM_KEY_GROUP");
+    });
+
+    it("should customize styling", async () => {
```

Comment:
suggestion: testing custom styles is done with vrt so these tests are redundant (unless there's a specific reason you're testing them here too)

Reply:
No there is no particulart reason. I just looked at other test files. The main one was to check element name changes which is done above. I'll actually merge these two to make sure element name changes make the style changes too

Reply:
yeah you'll see it in some old test files because we used to do it that way before we realized it was pointless bc the style tests happen in vrt